cypress/e2e/inventory/search/browse-search-subjects.cy.js

```diff
@@ -0,0 +1,46 @@
+import permissions from '../../../support/dictionary/permissions';
+import TopMenu from '../../../support/fragments/topMenu';
+import testTypes from '../../../support/dictionary/testTypes';
+import DevTeams from '../../../support/dictionary/devTeams';
+import Users from '../../../support/fragments/users/users';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import BrowseSubjects from '../../../support/fragments/inventory/search/browseSubjects';
+
+describe('Subject Browse', () => {
+  const testData = {
+    subjectName: 'Rock music--1961-1970',
```

Comment:
There is no guarantee that a record with this subject value will be available on environment when testing. Could you please add steps to create a new instance with certain subject and then search for it, or get subjects via API to know what is available?

Reply:
Added api function to create the testdata needed